libvirt-glib.rb

```diff
@@ -0,0 +1,105 @@
+class LibvirtGlib < Formula
+  desc "GLib and GObject mapping for libvirt"
+  homepage "http://www.libvirt.org"
+  url "http://libvirt.org/sources/glib/libvirt-glib-0.2.3.tar.gz"
+  sha256 "c3b11aa34584b3416148b36bb005505e461830c73c9fd2f40143cca3309250dd"
+
+  depends_on "gettext" => :build
+  depends_on "intltool" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "libvirt"
+
+  depends_on "gobject-introspection" => :optional
+  depends_on "vala" => :optional
+
+  # work around ld not understanding --version-script argument
+  # upstream bug: https://bugzilla.redhat.com/show_bug.cgi?id=1304981
+  patch :DATA
```

Comment:
If that has been accepted by upstream, then we don't need this, do we? So why not "for libvirt-glib 0.2.3 and maybe a few furture releases, we need this", and drop the patch when upstream has finally included a fix for Mac OS X. You must know it could take years before the patch for a unsupported platform makes its way into upstream trunk.